components/expression/expression.js

```diff
@@ -30,6 +30,24 @@ genotet.ExpressionView = function(viewName, params) {
   $(this.container).on('genotet.ready', function() {
     this.loader.load(params.matrixName, params.geneRegex, params.condRegex);
   }.bind(this));
+
+  $(this.renderer)
+    .on('genotet.cellHover', function(event, data) {
+      var cellSelection = d3.select(data[0]);
+      cellSelection.style('stroke', 'white');
```

Comment:
Please do not style here. Style is the job of renderer, so the code needs to be in renderer.js instead.